src/odig.mli

```diff
@@ -0,0 +1,11 @@
+type t
+
+type of_opam_error = Odig_not_installed
```

Comment:
I don't think we need this error type. We don't treat in any special way and just convert it to a message. Might as well do it as early as possible.